res/database/Default/database_news.xml

```diff
@@ -8701,95 +8610,24 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
+				<de>Nach historischem Vorbild wollen ${swedenandnorway} eine Union bilden. Auch ${finland} würde sich der neuen USL (Union Skandinavischer Länder) anschließen.</de>
+				<en>Following historical precedent, ${swedenandnorway} want to form a union. ${finland} also would join the new USN (Union of Scandinavian Nations).</en>
 			</description>
-			<variables>
-				<i>0|1|2|3|4|5|6|7|8|9</i>
-				<swedenandnorway_0>
-					<de>Schweden und Norwegen</de>
-					<en>Sweden and Norway</en>
-				</swedenandnorway_0>
-				<swedenandnorway_1>
-					<de>Dänemark und Schweden</de>
-					<en>Denmark and Sweden</en>
-				</swedenandnorway_1>
-				<swedenandnorway_2>
-					<de>Finnland und Schweden</de>
-					<en>Finnland and Sweden</en>
-				</swedenandnorway_2>
-				<swedenandnorway_3>
-					<de>Norwegen und Dänemark</de>
-					<en>Norway and Denmark</en>
-				</swedenandnorway_3>
-				<swedenandnorway_4>
-					<de>Norwegen und Finnland</de>
-					<en>Norway and Finland</en>
-				</swedenandnorway_4>
-				<swedenandnorway_5>
-					<de>Dänemark und Finnland</de>
-					<en>Denmark and Finland</en>
-				</swedenandnorway_5>
-				<swedenandnorway_6>
-					<de>Schweden und Island</de>
-					<en>Sweden and Iceland</en>
-				</swedenandnorway_6>
-				<swedenandnorway_7>
-					<de>Island und Dänemark</de>
-					<en>Iceland and Denmark</en>
-				</swedenandnorway_7>
-				<swedenandnorway_8>
-					<de>Island und Norwegen</de>
-					<en>Iceland and Norway</en>
-				</swedenandnorway_8>
-				<swedenandnorway_9>
-					<de>Finnland und Island</de>
-					<en>Finland and Iceland</en>
-				</swedenandnorway_9>
-				<finland_0>
-					<de>Finnland|Dänemark|Island</de>
-					<en>Finland|Denmark|Iceland</en>
-				</finland_0>
-				<finland_1>
-					<de>Norwegen|Finnland|Island</de>
-					<en>Norway|Finland|Iceland</en>
-				</finland_1>
-				<finland_2>
-					<de>Norwegen|Dänemark|Island</de>
-					<en>Norway|Denmark|Iceland</en>
-				</finland_2>
-				<finland_3>
-					<de>Schweden|Finnland|Island</de>
-					<en>Sweden|Finland|Iceland</en>
-				</finland_3>
-				<finland_4>
-					<de>Schweden|Dänemark|Island</de>
-					<en>Sweden|Denmark|Iceland</en>
-				</finland_4>
-				<finland_5>
-					<de>Schweden|Norwegen|Island</de>
-					<en>Sweden|Norway|Iceland</en>
-				</finland_5>
-				<finland_6>
-					<de>Dänemark|Norwegen|Finnland</de>
-					<en>Denmark|Norway|Finland</en>
-				</finland_6>
-				<finland_7>
-					<de>Norwegen|Schweden|Finnland</de>
-					<en>Norway|Sweden|Finland</en>
-				</finland_7>
-				<finland_8>
-					<de>Schweden|Finnland|Dänemark</de>
-					<en>Sweden|Finland|Denmark</en>
-				</finland_8>
-				<finland_9>
-					<de>Norwegen|Schweden|Dänemark</de>
-					<en>Norway|Sweden|Denmark</en>
-				</finland_9>
+			<variables> <!-- Select one of five Scandinavian countries, then select 2 of the remaining 4; maybe there's a simpler way to do this. -->
```

Comment:
You could create csv entries for each country followed by the others (still some duplication, but possibly better than now).
For selecting which other countries are used you create a variable <others>0;1|0;2|...|2;3</others> to determine disjoint indexes and split them to access the remaining countries.

Reply:
Another option would be just one csv entry with all countries and then a "selection" alternative <indexes>0;1;2|0;1;3|0;1;4|0;2;1|...</indexes> for a disjoint set of countries.

Reply:
ah yeah, option 2 is elegant, thanks